backend/api/v1/org_policy_service.go

```diff
@@ -1024,6 +1038,16 @@ func convertToBackupPlanPolicyPayload(policy *v1pb.BackupPlanPolicy) (*api.Backu
 	}, nil
 }
 
+func convertToV1RolloutPolicyPayload(payloadStr string) (*v1pb.Policy_RolloutPolicy, error) {
+	p := &v1pb.RolloutPolicy{}
```

Comment:
Still keep v1pb and storepb separate? More code but make meaningful difference in the future.